airbyte-integrations/connectors/source-amazon-mws/integration_tests/configured_catalog.json

```diff
@@ -0,0 +1,178 @@
+              "type": "string",
+              "title": "item-price",
+              "description": ""
+            },
+            "item-tax": {
+              "type": "string",
+              "title": "item-tax",
+              "description": ""
+            },
+            "shipping-price": {
+              "type": "string",
+              "title": "shipping-price",
+              "description": ""
+            },
+            "shipping-tax": {
+              "type": "string",
+              "title": "shipping-tax",
+              "description": ""
+            },
+            "gift-wrap-price": {
+              "type": "string",
+              "title": "gift-wrap-price",
+              "description": ""
+            },
+            "gift-wrap-tax": {
+              "type": "string",
+              "title": "gift-wrap-tax",
+              "description": ""
+            },
+            "item-promotion-discount": {
+              "type": "string",
+              "title": "item-promotion-discount",
+              "description": ""
+            },
+            "ship-promotion-discount": {
+              "type": "string",
+              "title": "ship-promotion-discount",
+              "description": ""
+            },
+            "ship-city": {
+              "type": "string",
+              "title": "ship-city",
+              "description": ""
+            },
+            "ship-state": {
+              "type": "string",
+              "title": "ship-state",
+              "description": ""
+            },
+            "ship-postal-code": {
+              "type": "string",
+              "title": "ship-postal-code",
+              "description": ""
+            },
+            "ship-country": {
+              "type": "string",
+              "title": "ship-country",
+              "description": ""
+            },
+            "promotion-ids": {
+              "type": "string",
+              "title": "promotion-ids",
+              "description": ""
+            },
+            "is-business-order": {
+              "type": "string",
+              "title": "is-business-order",
+              "description": ""
+            },
+            "purchase-order-number": {
+              "type": "string",
+              "title": "purchase-order-number",
+              "description": ""
+            },
+            "price-designation": {
+              "type": "string",
+              "title": "price-designation",
+              "description": ""
+            },
+            "fulfilled-by": {
+              "type": "string",
+              "title": "fulfilled-by",
+              "description": ""
+            },
+            "is-sold-by-ab": {
+              "type": "string",
+              "title": "is-sold-by-ab",
+              "description": ""
+            }
+          }
+        },
+        "supported_sync_modes": ["full_refresh", "incremental"],
+        "source_defined_cursor": false,
+        "default_cursor_field": []
```

Comment:
based on my reading of the API, a few comments: 
1. The API defines the cursor value as being either created_at or updated_at date. I think we should use `updated_at` at as it includes both creations (when an object is created, updated_at = created_at so using updated_at still finds new objects)
2. since the user will not be configuring the cursor, `source_defined_cursor` should be output as true